frontend/src/metabase-lib/column_types.ts

```diff
@@ -38,3 +39,10 @@ export const isTime: TypeFn = TYPES.time_QMARK_;
 export const isTitle: TypeFn = TYPES.title_QMARK_;
 export const isURL: TypeFn = TYPES.URL_QMARK_;
 export const isZipCode: TypeFn = TYPES.zip_code_QMARK_;
+
+export function isCompatibleType(
+  column1: ColumnMetadata | DatasetColumn,
```

Comment:
@uladzimirdev it's only `ColumnMetadata` now

Reply:
[63f9ae6](https://github.com/metabase/metabase/pull/37438/commits/63f9ae6b4b6112a1a11544c44874d77dfeac0677)